virttest/utils_disk.py

```diff
@@ -445,7 +445,15 @@ class of guest disk using guestfs lib to do some operation(like read/write)
     on guest disk:
     """
 
-    def __init__(self, disk):
+    def __init__(self, disk, backend='direct'):
+        """
+        :params backend: Let libguestfs creats/connects to backend daemon
```

Comment:
1. %s/creats/creates
2. can we add docstring for disk too

Reply:
Modified it, thanks.